lucene/core/src/java/org/apache/lucene/index/StandardDirectoryReader.java

```diff
@@ -169,7 +189,10 @@ static StandardDirectoryReader open(
    * @lucene.internal
    */
   public static DirectoryReader open(
-      Directory directory, SegmentInfos infos, List<? extends LeafReader> oldReaders)
+      Directory directory,
```

Comment:
Are we handling the near-real-time reader (`IndexWriter.getReader(...)`) path here too?

Reply:
It looks like  `IndexWriter.getReader(...)` path is covered by a previous `open` method on line 123.